src/sentry/models/organization.py

```diff
@@ -718,9 +721,15 @@ def organization_absolute_url(
     return "".join(parts)
 
 
+def delete_org_refs_at_control_silo(instance: Organization, **kwargs):
+    # Send RPC requests to the control silo
+    Organization.remove_organization_mapping(instance)
+    organizationmember_mapping_service.delete_by_org_id(organization_id=instance.id)
```

Comment:
Wouldn't we be able to take these actions in Control Silo when the tombstone for Organization is processed?  Currently `OrganizationMemberMapping.organization_id` is marked as a HybridCloudForeignKey.